### PR TITLE
Added round edges to "WHAT WE OFFER" section #204

### PR DESCRIPTION
**Pull request by The-Arshad-Khan**
## 🛠️ Fixes Issue

Closes #

## 👨‍💻 Changes proposed
Added Round Edges To "WHAT WE OFFER" SECTION.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
image

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -73,6 +73,11 @@ html {
 
 .welcome .item {
   text-align: center;
+  border-radius: 20px;
+  -webkit-border-radius: 20px;
+  -moz-border-radius: 20px;
+  -ms-border-radius: 20px;
+  -o-border-radius: 20px;
```
Comment (sakshamgurbhele) [inline css/main.css:80]: lgtm

**Review by sakshamgurbhele — CHANGES_REQUESTED** [review]
Hey @The-Arshad-Khan, Can you delay the transition for 1-2 seconds, it will feel better I think.

File: css/main.css (hunk 1 above)
Comment (The-Arshad-Khan) [reply css/main.css:80]: @sakshamgurbhele On it;

Comment (The-Arshad-Khan) [reply css/main.css:80]: and Done!

**Comment by The-Arshad-Khan** [comment]
@sakshamgurbhele Kindly merge my PR!

**Comment by sakshamgurbhele** [comment]
@The-Arshad-Khan Thank You for Contributing, Merging it 🚀

**Review by sakshamgurbhele — COMMENTED** [review]
cool 👍🏼

**Pull request merged**